github/authorizations.go

```diff
@@ -343,8 +343,14 @@ func (s *AuthorizationsService) Revoke(ctx context.Context, clientID string, tok
 // tokens an application has generated for the user.
 //
 // GitHub API docs: https://developer.github.com/v3/oauth_authorizations/#list-your-grants
-func (s *AuthorizationsService) ListGrants(ctx context.Context) ([]*Grant, *Response, error) {
-	req, err := s.client.NewRequest("GET", "applications/grants", nil)
+func (s *AuthorizationsService) ListGrants(ctx context.Context, opt *ListOptions) ([]*Grant, *Response, error) {
+	u := "applications/grants"
```

Comment:
Since the value of `u` here is just a constant, inline it into the next line:

```Go
u, err := addOptions("applications/grants", opt)
if err != nil {
	return nil, nil, err
}
```

It makes sense to keep it separate only when it's using `fmt.Sprintf`.

See existing code for precedent.